pkg/controllers/v1alpha1/dataload/implement.go

```diff
@@ -331,10 +300,49 @@ func (r *DataLoadReconcilerImplement) reconcileExecutingDataLoad(ctx cruntime.Re
 				if jobCondition.Type == batchv1.JobFailed {
 					dataloadToUpdate.Status.Phase = common.PhaseFailed
 				} else {
-					dataloadToUpdate.Status.Phase = common.PhaseComplete
+
+					//  Check existence of the targetPath in alluxio && check dataLoad completely by pod's logs.
```

Comment:
I am a little confused about the logic here. The log check happens when `jobCondition.Type != batchv1.JobFailed`, which means either the DataLoad pod is still running or it already succeeds. In either of the cases, I think we won't find logs like `path does not exists"